utility/src/main/java/org/oppia/android/util/parser/html/LiTagHandler.kt

```diff
@@ -45,7 +45,9 @@ class LiTagHandler(
     when (tag) {
       CUSTOM_LIST_UL_TAG -> lists.pop()
       CUSTOM_LIST_OL_TAG -> lists.pop()
-      CUSTOM_LIST_LI_TAG -> lists.peek().closeItem(output, indentation = lists.size - 1)
+      CUSTOM_LIST_LI_TAG ->
+        if (lists.isNotEmpty())
```

Comment:
This needs braces per https://developer.android.com/kotlin/style-guide#braces.

Reply:
Done.